indra/cmake/Copy3rdPartyLibs.cmake

```diff
@@ -61,6 +61,15 @@ if(WINDOWS)
         uriparser.dll
         )
 
+    # ICU4C (same filenames for 32 and 64 bit builds)
+    set(release_files ${release_files} icudt48.dll)
+    set(release_files ${release_files} icuin48.dll)
+    set(release_files ${release_files} icuio48.dll)
+    set(release_files ${release_files} icule48.dll)
+    set(release_files ${release_files} iculx48.dll)
+    set(release_files ${release_files} icutu48.dll)
+    set(release_files ${release_files} icuuc48.dll)
+
```

Comment:
I thought we determined that only the 2 largest ICU4C files were needed but since the others are tiny by comparison, and it's notoriously hard to know if a DLL is needed or not, I think this is okay unless anyone else cares.

Reply:
It copies all the DLLs over to build-vcXXX-YY\sharedlibs\Release but then only copies the ones that are actually references by the viewer executable into the final build folder (this is done by the DeploySharedLibs cmake step if I remember correctly.

After just building the viewer on Release with PACKAGE=ON the sharedlibs\Release folder contains all:
![image](https://user-images.githubusercontent.com/42094019/223278568-18113c65-3b4c-449c-a5de-c8d461304379.png)

But then the build folder (and the eventual end-user install folder) only contains the ones needed to run the viewer:
![image](https://user-images.githubusercontent.com/42094019/223278635-098f27a8-6abe-442b-a408-d80e0b6831a0.png)

That said, out of all IC4C we only need a singular very simple function so switching to static linking could remove the 18.6Mb of DLL files with - guesstimating - 100Kb of statically linked code so we can still optimize that one.